js/main.js

```diff
@@ -0,0 +1,55 @@
+let books = [];
+
+function showBook(book) {
+  let element;
+  const div = document.createElement('div');
+  element = document.createElement('p');
+  element.appendChild(document.createTextNode(book.title));
+  div.appendChild(element);
+  element = document.createElement('p');
+  element.appendChild(document.createTextNode(book.author));
+  div.appendChild(element);
+  element = document.createElement('button');
+  element.setAttribute('type', 'button');
+  element.appendChild(document.createTextNode('Remove'));
+  element.addEventListener('click', () => {
+    // remove the book from the screen
+    div.remove(book.id);
+    // remove the book from the local storage
+    const newArray = books.filter((book2) => book2.id !== book.id);
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20remove%20a%20book%20from%20the%20collection%20(hint%3A%20you%20can%20use%20the%20array%20filter()%20method).), Please create a separate function to remove a book from book collection using [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method then you can call it here in `showBook` function.